dspace-api/src/main/java/org/dspace/statistics/SolrLoggerUsageEventListener.java

```diff
@@ -53,14 +57,27 @@ public void receiveEvent(Event event) {
                                                    currentUser);
                     }
                 } else if (UsageEvent.Action.SEARCH == ue.getAction()) {
-                    UsageSearchEvent usageSearchEvent = (UsageSearchEvent) ue;
-                    //Only log if the user has already filled in a query !
-                    if (!CollectionUtils.isEmpty(((UsageSearchEvent) ue).getQueries())) {
-                        solrLoggerService.postSearch(ue.getObject(), ue.getRequest(), currentUser,
-                                                     usageSearchEvent.getQueries(), usageSearchEvent.getRpp(),
-                                                     usageSearchEvent.getSortBy(),
-                                                     usageSearchEvent.getSortOrder(), usageSearchEvent.getPage(),
-                                                     usageSearchEvent.getScope());
+                    if (ue instanceof UsageSearchEvent) {
```

Comment:
This seems to be an instanceof implementation because the old UsageSearchEvent can contain multiple queries (which is weird). I would recommend to use UsageSearchEvent instead, remove the instanceof and fix UsageSearchEvent to contain a single query